pkgs/desktops/xfce/panel-plugins/xfce4-vala-panel-appmenu-plugin/default.nix

```diff
@@ -45,5 +45,6 @@ stdenv.mkDerivation rec {
     description = "Global Menu applet for XFCE4";
     license = licenses.lgpl3;
     maintainers = with maintainers; [ jD91mZM2 ];
+    meta.broken = true;
```

Comment:
tbh, this is just not being maintained by it's maintainer in nixpkgs.
There's releases that probably fixed it https://github.com/rilian-la-te/vala-panel-appmenu/releases.